SenderMail/src/com/company/Main.java

```diff
@@ -0,0 +1,114 @@
+package com.company;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class Main {
+    //Permet de récuperer la date d'aujourdhui.
+    public static String today() {
+        String format = "YYYY/MM/dd";
+        java.text.SimpleDateFormat formatter = new java.text.SimpleDateFormat( format );
+        java.util.Date date = new java.util.Date();
+        return  formatter.format(date) ;
+    }
+
+
+    public static void main(String[] args) {
+        ArrayList mailsBirthday = new ArrayList();
+        int numberBirthday = 0;
+        Scanner sc = new Scanner(System.in);
+        boolean quitter=true;
+        while(quitter) {
+            System.out.println("a.Recherche si anniversaire à souahiter aujourd'hui. \n");
+            System.out.println("b.Liste des mails d'anniversaires à envoyer. \n");
+            System.out.println("c.Envoie du ou des mails.");
+            String str = sc.nextLine();
+
+            switch (str) {
+
+                case "a":
+                    try {
+
+                        InputStream flux = new FileInputStream("humans.txt");
+                        InputStreamReader lecture = new InputStreamReader(flux);
+                        BufferedReader buff = new BufferedReader(lecture);//Stocke les donnes du fichier humans.txt dans un buffer
+
+                        String line;
+                        String today = today();
+
+                        System.out.println("Date d'aujourd'hui: " + today);
+                        System.out.println("------------------------------------------------------------");
+
+                        String[] arrOfToday = today.split("/");//Separe la date d'aujourdhui et la stocke dans un tableau de string.
+                        String todayMonth = arrOfToday[1];
+                        String todayDay = arrOfToday[2];
```

Comment:
Java propose des méthodes pour récupérer le mois et le jours de la date actuelle.
```suggestion
                        Calendar cal = Calendar.getInstance();
                        int month = cal.get(Calendar.MONTH);
		        int day = cal.get(Calendar.DAY_OF_MONTH);
```

Reply:
Merci du conseil